dss/dynamodb/__init__.py

```diff
@@ -3,53 +3,54 @@
 from dss.util.aws.clients import dynamodb as db  # type: ignore
 
 
-def put_item(table: str, value: str, hash_key: str, sort_key: str):
+def format_item(value, hash_key, sort_key):
```

Comment:
Consider making this method private, and making `value` and `sort_key` keywords.

Reply:
Alright!

Reply:
Made private and separated the function into 2 instead for the value keyword.